### DISCUSSION TITLE
Component for "Fork in the Road"?

### DISCUSSION THREAD

**Discussion by jme783** [twilio-labs/paste#1378] [Q&A] [answered]
Is there a recommended paste component for having a user choose from two main pathways — e.g. a “fork in the road” beyond just a simple radio button?

Context: for the .org program, we will have 2 slightly different signup processes for nonprofits vs. other organization types (e.g. social enterprises) and need to have the user self-identify to render the right flow. I was thinking maybe using 2 Cards but definitely open to suggestions from the experts!

**Comment by zahnster** [comment, marked as answer]
Hi @jme783,

Paste doesn't have an out of the box component to handle this, but likely can provide a solution. 

Do you have any visual mockups of the interface you are hoping to achieve? Depending on what action happens after your "fork in the road" action is taken, we may have different recommendations for you. High level, your initial thoughts seem on track (having two cards, each with an action within).

**Reply by zahnster** [reply]
Discussed on Slack with Jeff and referred him to the [Visual Picker](https://lightningdesignsystem.com/components/visual-picker/) component that exists in Salesforce's Lightning Design System for inspiration on more visually complex selection UI elements.
